src/elements/kw-social-comment/kw-social-comment.html

```diff
@@ -148,37 +186,83 @@ <h3 shown$="[[!comments.length]]">Be the first to comment!</h3>
             </div>
         </div>
         <p>Remember to be nice</p>
-        <template is="dom-repeat" items="{{comments}}" as="comment">
-            <div class$="comment [[_computeOptimisticClass(comment.optimistic)]]">
-                <div class="comment-avatar">
-                    <iron-image preload placeholder="/assets/icons/shares/commentAvatar.svg" fade sizing="contain" src="{{comment.author.avatar.urls.circle}}"></iron-image>
-                </div>
-                <div class="content">
-                    <p><span class="bold">{{comment.author.username}}</span><span inner-h-t-m-l="[[_lb(comment.text)]]"></span></p>
-                    <date-util format="H:mm , dd Mon yyyy" date="[[_createDate(comment.date_created)]]"></date-util>
-                </div>
-            </div>
-        </template>
+        <iron-list items="[[comments]]" as="comment" hidden$="[[comment]]">
+                <template>
+                    <div class$="comment [[_computeOptimisticClass(comment.optimistic)]]">
```

Comment:
Total niggle, and I know it was pre-existing, but what do we mean by `optimistic`? Is there a more descriptive word that we can use?